From 7baf499d212c192e841221453f457dfd7c3a27a3 Mon Sep 17 00:00:00 2001
From: "Gopal, Puunithaaraj" <puunithaaraj.gopal@intel.com>
Date: Sat, 4 May 2019 00:57:45 +0800
Subject: [PATCH 1106/1214] media: ici: Fixed bytesperline calculation for
 packed format

Fixed bytesperline calculation for packed format during front end capture

Change-Id: I755dcf094bd4f1f04725e26bc8aa47f724562b6e
Tracked-On: PKT-2287
Signed-off-by: Gopal, Puunithaaraj <puunithaaraj.gopal@intel.com>
---
 drivers/media/pci/intel/ici/ici-isys-stream.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/media/pci/intel/ici/ici-isys-stream.c b/drivers/media/pci/intel/ici/ici-isys-stream.c
index a01e135..4523ce9 100644
--- a/drivers/media/pci/intel/ici/ici-isys-stream.c
+++ b/drivers/media/pci/intel/ici/ici-isys-stream.c
@@ -1015,10 +1015,10 @@ const struct ici_isys_pixelformat
 			 mpix->ffmt.width * DIV_ROUND_UP(pfmt->bpp,
 			 BITS_PER_BYTE);
 	else
-		mpix->pfmt.plane_fmt[0].bytesperline = DIV_ROUND_UP(
-			as->line_header_length + as->line_footer_length
-			+ (unsigned int)mpix->ffmt.width * pfmt->bpp,
-			BITS_PER_BYTE);
+		mpix->pfmt.plane_fmt[0].bytesperline =
+			DIV_ROUND_UP((unsigned int)mpix->ffmt.width *
+				pfmt->bpp,
+				BITS_PER_BYTE);
 
 	mpix->pfmt.plane_fmt[0].bytesperline =
 		ALIGN(mpix->pfmt.plane_fmt[0].bytesperline,
-- 
2.7.4

